src/components/modals/AddAccounts/steps/03-step-import.js

```diff
@@ -14,9 +15,22 @@ import Button from 'components/base/Button'
 import AccountsList from 'components/base/AccountsList'
 import IconExclamationCircleThin from 'icons/ExclamationCircleThin'
 import TranslatedError from '../../../TranslatedError'
+import Spinner from '../../../base/Spinner'
 
 import type { StepProps } from '../index'
 
+const LoadingRow = styled(Box).attrs({
+  horizontal: true,
+  borderRadius: 1,
+  px: 3,
+  align: 'center',
+  justify: 'center',
+  mt: 1,
```

Comment:
more a generic consideration, but imo the margins should be set contextually and not inside components
using `<LoadingRow mt={1} />` make it more reusable, because component is not leaking any margin itself

Reply:
but the component is in place here.. also i just copy what was before in Row

Reply:
pushing the logic more far, that means we never want to do `styled(Box).attr` right?

Reply:
no, it's only for margins that should not be part of the component style, but only when we _use_ the component in a page